### PR TITLE
Twitter api

### PR DESCRIPTION
**Pull request by derekjou**

### CODE REVIEW

**Comment by ncen5293** [comment]
can we wait until the interface is merged into dev branch so that we can get mentions on a uniform way

**Comment by derekjou** [comment]
> can we wait until the interface is merged into dev branch so that we can get mentions on a uniform way

Yeah, sure!

File: client/src/pages/Dashboard.js
Comment (aeciorc) [inline client/src/pages/Dashboard.js]: Can we have a wrapper for axios that performs that check for every authenticated request?

**Review by Ahana15 — APPROVED** [review]
looks good

**Pull request merged**
